main_routers/memory_router.py

```
@@ -298,8 +347,11 @@ async def update_catgirl_name(request: Request):
         memory_dir = Path(cm.memory_dir)
         
         # Construct and validate file paths
-        old_filename = f'recent_{old_name}.json'
-        new_filename = f'recent_{new_name}.json'
+        old_filename = f'recent_{old_name}.xml'
+        new_filename = f'recent_{new_name}.xml'
+        # 向后兼容：如果XML文件不存在，尝试JSON文件
+        old_json = f'recent_{old_name}.json'
+        new_json = f'recent_{new_name}.json'
```

Ruff (0.14.8)

Local variable `new_json` is assigned to but never used

Remove assignment to unused variable `new_json`

(F841)